main.rb

```diff
@@ -0,0 +1,36 @@
+require 'app'
```

Comment:
- [ ] Please use `require_relative` instead of `require` as you did in the other files because it's being used to include the files from the same directory.

Reply:
done